### ISSUE TITLE
Engineers: Attend breakout session 

### ISSUE THREAD

**Issue by Justin-Pickett** [department-of-veterans-affairs/va.gov-team#964]
Engineers please check the button below and comment "done" when this is completed. 

- [ ] Engineers: Attend breakout session

**Comment by erikphansen** [comment]
Was this the VSA engineering onboarding meeting held Monday Aug 5? Or is breakout session something else still to come?

**Comment by Justin-Pickett** [comment]
Yes @erikphansen  this was the initial breakout session for engineering Aug. 5th.

**Comment by erikphansen** [comment]
done

**Comment by Justin-Pickett** [comment]
@lihanli and @mojaray2k please comment done on this ticket.

**Comment by mojaray2k** [comment]
Done
